docs/tools/firmware.sh

```diff
@@ -117,74 +117,69 @@ create_rpm () {
-
-	echo -e "\n%post" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "modprobe -r brcmfmac_wcc || true" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "modprobe -r brcmfmac || true" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "modprobe brcmfmac || true" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "modprobe -r hci_bcm4377 || true" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "modprobe hci_bcm4377 || true" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-
-	echo -e "\n%files" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-	echo "/usr/lib/firmware/brcm/*" >> $HOME/rpmbuild/SPECS/apple-firmware.spec
-
-	# Set OS to Linux
-	if [ -f "$HOME/.rpmrc" ]
-	then
-		echo -e "\nBacking up existing .rpmrc"
-		cp ${verbose} $HOME/.rpmrc $HOME/.rpmrc_orj
-	fi
-	echo "buildostranslate: Darwin: Linux" > $HOME/.rpmrc
+	cat <<- EOF > $HOME/rpmbuild/SPECS/apple-firmware.spec
+		Name:       apple-firmware
+		Version:    ${ver}
+		Release:    1
+		Summary:    Wi-Fi and Bluetooth firmware for T2 Macs
+		License:    Proprietary
+		BuildArch: noarch
+
+		Source1: firmware.tar
+
+		%description
+		Wi-Fi and Bluetooth firmware for T2 Macs
+
+		%prep
+		tar -xf %{SOURCE1}
+
+		%build
+
+		%install
+		mkdir -p %{buildroot}/usr/lib/firmware/brcm
+		install -m 644 * %{buildroot}/usr/lib/firmware/brcm
+
+		%posttrans
+		modprobe -r brcmfmac_wcc || true
+		modprobe -r brcmfmac || true
+		modprobe brcmfmac || true
+		modprobe -r hci_bcm4377 || true
+		modprobe hci_bcm4377 || true
+
+		%files
+		/usr/lib/firmware/brcm/*
+	EOF
 
 	# Build
-	# For some reason bash doesn't like rpmbuild. Use zsh.
 	if [[ ${verbose} = -v ]]
 	then
-		/bin/zsh -c "rpmbuild -bb $HOME/rpmbuild/SPECS/apple-firmware.spec"
+		rpmbuild -bb --define '_target_os linux' $HOME/rpmbuild/SPECS/apple-firmware.spec
 	else
-		/bin/zsh -c "rpmbuild -bb $HOME/rpmbuild/SPECS/apple-firmware.spec >/dev/null 2&>/dev/null || echo \"Failed to make rpm package. Run the script with -v to get logs.\""
+		rpmbuild -bb --define '_target_os linux' $HOME/rpmbuild/SPECS/apple-firmware.spec >/dev/null 2>&1 || echo "Failed to make rpm package. Run the script with -v to get logs."
 	fi
 
 	# Copy and Cleanup
-	cp ${verbose} $HOME/rpmbuild/RPMS/x86_64/apple-firmware-${ver}-1.x86_64.rpm $HOME/Downloads
+	cp ${verbose} $HOME/rpmbuild/RPMS/noarch/apple-firmware-${ver}-1.noarch.rpm $HOME/Downloads
 	echo -e "\nCleaning up"
-	sudo rm -r ${verbose} $HOME/rpmbuild
-	sudo rm $HOME/.rpmrc
-	if [ -f $HOME/.rpmrc_orj ]
-	then
-		echo -e "\nRestoring backed up .rpmrc"
-		mv $HOME/.rpmrc_orj $HOME/.rpmrc
-	fi
+	rm -r ${verbose} $HOME/rpmbuild
 
-	echo -e "\nRpm package apple-firmware-${ver}-1.x86_64.rpm has been saved to Downloads!"
-	echo "Copy it to Linux and install using rpm."
+	echo -e "\nRpm package apple-firmware-${ver}-1.noarch.rpm has been saved to Downloads!"
+	echo "Copy it to Linux and install it with \`sudo dnf install --disablerepo=* /path/to/apple-firmware-${ver}-1.noarch.rpm\`"
```

Comment:
Instead of:

Copy it to Linux and install it with `sudo dnf install --disablerepo=* /path/to/apple-firmware-14.4.1-1.noarch.rpm` 

Maybe this looks better:

Copy it to Linux and install it by running the following in the Linux terminal:

sudo dnf install --disablerepo=* /path/to/apple-firmware-14.4.1-1.noarch.rpm